app/views/practices/show/introduction/_introduction.html.erb

```diff
@@ -63,7 +63,7 @@
       </div>
       <%# practice summary %>
       <p class="desktop:grid-col-10 grid-col-12 margin-bottom-2 font-sans-lg line-height-sans-6">
-        <%= @practice.summary %>
+        <%= safe_join(h(@practice.summary).split("\r\n"), tag(:br)) %>
```

Comment:
let me know if you guys think this should be a named method... like... `render_line_breaks(string)`
i thought it was simple enough to place as is, but i'm open to thoughts :smile:

Reply:
this feels good for now... maybe if we are using it more frequently we could make a method for it 🤔

Reply:
Yeah - i think it's fine as is.  Should we wrap all text area elements with this going forward?

Reply:
@bradjohnson92008 no, not yet.
when alex requests them, which may be soon, hah.